### PR TITLE
fix: [PL-22728]: Moving HARNESS_ENABLE_NG_AUTH_UI to dev only config

### PR DESCRIPTION
**Pull request by avinashmadhwani02**

### CODE REVIEW

**Review by abhinavrastogi-harness — DISMISSED** [review]

File: src/index.html [hunk 1]
```diff
@@ -48,6 +47,7 @@
     </script>
     <% } %>
     <!-- Enabling ng-auth-ui -->
+    <!-- dev only flag -->
     <% if (HARNESS_ENABLE_NG_AUTH_UI) { %>
```
Comment (mayankVermaHarness) [inline src/index.html:51]: how does `HARNESS_ENABLE_NG_AUTH_UI` have its value hydrated?
IIRC, there should be reference of the same variable in `HTMLWebpackPlugin` in webpack file

Comment (avinashmadhwani02) [reply src/index.html:51]: There's a variable in Define Plugin, right? the same way we are using __DEV__

File: configs/webpack.prod.js [hunk 2]
```diff
@@ -37,7 +37,8 @@ const config = {
   plugins: [
     new webpack.DefinePlugin({
       __DEV__: false,
-      __BUGSNAG_RELEASE_VERSION__: JSON.stringify(packageJson.version)
+      __BUGSNAG_RELEASE_VERSION__: JSON.stringify(packageJson.version),
+      HARNESS_ENABLE_NG_AUTH_UI: false
```
Comment (abhinavrastogi-harness) [inline configs/webpack.prod.js:41]: will this force-disable the flag, even if the env variable is true in SaaS?

**Review by abhinavrastogi-harness — CHANGES_REQUESTED** [review]

File: configs/webpack.prod.js (hunk 2 above)
Comment (avinashmadhwani02) [reply configs/webpack.prod.js:41]: No this will not, HARNESS_ENABLE_NG_AUTH_UI will be replaced in index.html once the build is generated

**Review by abhinavrastogi-harness — APPROVED** [review]

**Review by mayankVermaHarness — APPROVED** [review]

**Pull request merged**